specification/2.0/README.md

```diff
@@ -3,13 +3,56 @@ Copyright 2014-2021 The Khronos Group Inc.
 SPDX-License-Identifier: CC-BY-4.0
 -->
 
-The glTF 2.0 Specification formerly hosted in this README.md has been moved
-to AsciiDoc markup. Published versions of the Specification are now located
-in the [glTF Registry](https://www.khronos.org/registry/glTF). The
-underlying AsciiDoc markup is in the `main` branch of the [glTF
-Specification Repository](https://www.github.com/KhronosGroup/glTF).
-
-A preview of the rendered output of the specification can be seen by
-opening the [`Specification.adoc`](Specification.adoc) file, but note that
-this preview does not include the properties reference and JSON schema
-reference, and will not render embedded mathematical equations correctly.
+![glTF 2.0](../figures/gltf.png)
+
+# The glTF 2.0 Specification
+
+The full specification is available in two forms:
+
+- [Specification as HTML](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html) - for viewing in a browser
+- [Specification as PDF](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.pdf) - for viewing as a document
+
+## Extensions
+
+Features not covered by the core glTF specification may be contributed by way of extensions. See the [glTF Extension Registry](https://github.com/KhronosGroup/glTF/tree/main/extensions) for details.
+
+## Contributing
```

Comment:
There's also a repo-level [CONTRIBUTING](https://github.com/KhronosGroup/glTF/blob/main/CONTRIBUTING.adoc) file that is being linked under each Issue and PR form. Should it be updated with a link to this section?

Reply:
Added.